packages/verified-fetch/src/utils/dir-index-html.ts

```diff
@@ -165,7 +165,7 @@ export const dirIndexHtml = (dir: UnixFSEntry, items: UnixFSEntry[], { gatewayUR
     },
     listing: items.map((item) => {
       return {
-        size: item.size.toString(),
+        size: item.size?.toString() ?? '?',
```

Comment:
we no longer have the sizes.. we will need to either fetch them with stat or decide on a better default.